PROVISIONING.md

```diff
@@ -71,6 +73,11 @@ Use case: large scale deployments (where automation and security are major drive
 - Device reboots into newly installed software
   - Question: if the version is the same, can the entire process skip ? where this happens?
 
+Do we want to favor UEFI methods (like HTTPS boot) over others that require a client running in an OS, or a BMC (like sZTP)?
```

Comment:
I feel as if when we start to produce code for this, we would definitely want to prioritize the specific methods of booting. So I think it's less favoring and more prioritizing.